Lib/ufoLib/validators.py

```diff
@@ -48,13 +47,26 @@ def genericIntListValidator(values, validValues):
 		return False
 	valuesSet = set(values)
 	validValuesSet = set(validValues)
-	if len(valuesSet - validValuesSet) > 0:
+	if valuesSet - validValuesSet:
 		return False
 	for value in values:
 		if not isinstance(value, int):
 			return False
 	return True
 
+def genericIntSetValidator(values, validValues):
+	"""
+	Generic. (Added at version 3.)
+	"""
+	if not isinstance(values, (list, tuple)):
```

Comment:
i think I've already noted this.. why a function called `genericIntSetValidator` and then `values` is expected to be list or tuple?!